src/views/payment-sheet-views/card-view.js

```diff
@@ -49,10 +69,14 @@ CardView.prototype.initialize = function () {
     postalCodeFieldGroup.parentNode.removeChild(postalCodeFieldGroup);
   }
 
-  if (this.hasCardholderName) {
-    this._setupCardholderName(cardholderNameField);
-  } else {
-    cardholderNameField.parentNode.removeChild(cardholderNameField);
+  for (i = 0; i < this.extraInputs.length; i++) {
```

Comment:
Is there a reason to use a for loop instead of forEach?

Reply:
No, will update this